app/jobs/delete_old_alert_runs_job.rb

```diff
@@ -0,0 +1,7 @@
+class DeleteOldAlertRunsJob < ApplicationJob
+  queue_as :low
+
+  def perform
+    AlertRun.where(run_on: ...1.week.ago).destroy_all
```

Comment:
Having looked at historical alert run data, is it worth keeping a month of this?

Reply:
I would like to push for making the table as small as we can (and a week seems plenty given the code only checks for "today") so we can ensure a significant impact with the change.